setup.py

```diff
@@ -15,30 +15,56 @@
 
 
 # TODO: change this to something more stable when available.
-WEBHOST_URL = ('https://ci.appveyor.com/api/buildjobs/nheu483rq96ssbst'
-               '/artifacts/Functions.Binaries.2.0.12014-alpha.no-runtime.zip')
+WEBHOST_URL = ('https://ci.appveyor.com/api/buildjobs/k11e6ngq22n1kgfm'
+               '/artifacts/Functions.Binaries.2.0.12041-alpha.no-runtime.zip')
```

Comment:
Please use the latest version in prod https://ci.appveyor.com/project/appsvc/azure-webjobs-sdk-script-y8o14/build/2.0.12050/job/ykwva3bprej48cfa/artifacts